SWIPs/swip-message_to_honey.md

```diff
@@ -72,8 +104,7 @@ This SWIP is backward compatible as long as the price oracle quotes the same pri
 No test cases for this SWIP are provided at this moment
```

Comment:
in order to be consistent with the following sections, add `.` at the end of this section

Reply:
Changed!